packages/widgets-manager/src/grid.ts

```diff
@@ -85,12 +155,30 @@ export class GridManager {
     cell.apps = positionStrategy.mountStrategy().add(app, cell.apps);
   }
 
+  // instead of create a new cell with need to map the closet cell that match on at least one axis x or y
+  // the complete refactor it's going to be added in the next widget manager release
   _addToNewCell(app: App, id: string, positionStrategy: PositionStrategy) {
     if (positionStrategy.shouldAddNewPosition()) {
       this.grid.cells.push(this._createNewCell(id, app, positionStrategy.mountStrategy()));
     }
   }
 
+  _addToClosetCell(app: App, id: string) {
+    const splitPosition = id.split('-')
```

Comment:
I would have a `_findClosestCell` that just return the id (string) of the closest Cell

Reply:
Just an idea, maybe easier to refactor, not required